charts/kong/templates/service-kong-proxy.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   name: {{ template "kong.fullname" . }}-proxy
 spec:
   type: {{ .Values.kong.proxy.type }}
+  {{- if eq .Values.kong.proxy.type "LoadBalancer" }}
+  {{- if .Values.kong.proxy.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.kong.proxy.loadBalancerIP }}
```

Comment:
wondering we can merge these two if conditions, something like,
`{{- if and (eq .Values.kong.proxy.type "LoadBalancer") .Values.kong.proxy.loadBalancerIP }}`

Reply:
Hot damn, I looked for that syntax for something like half an hour.  Thanks!